pyproject.toml

```diff
@@ -15,10 +15,11 @@ converter = "converter.cli:cli"
 
 [tool.poetry.dependencies]
 python = ">=3.9.0,<4"
-typer = "^0.6.1"
+typer = {version = "^0.7.0", extras = ["all"]}
 deepdiff = "^5.8.1"
 fastapi = "^0.80.0"
 stringcase = "^1.2.0"
+pytest = "7.2.0"
```

Comment:
We don't want to add pytest as a dependency, rather keep it as a dev-dependency.